lib/libvcc/vcc_utils.c

```diff
@@ -285,6 +285,28 @@ Resolve_Sockaddr(struct vcc *tl,
 	ZERO_OBJ(rss, sizeof rss);
 }
 
+/*--------------------------------------------------------------------
+* Recognize boolean const "true" or "false"
+*/
+
+uint8_t
+vcc_BoolVal(struct vcc *tl)
+{
+	struct symbol* sym;
+
+	if (tl->t->tok != ID) {
+		Expect(tl, ID);
+		return (0);
+	}
```

Comment:
These lines are superfluous, `VCC_SymbolGet()` will fail just fine on its own.

Reply:
I agree it will fail, but with a simple assert, whereas this one shows a clearer error message and designates the problematic token in the VCL file (line number and position)

Reply:
I see, then we should also add coverage with `.expect_close = 1`. Good catch!